### ISSUE TITLE
about pretrain weight

### ISSUE THREAD

**Issue by Vyting-song** [Jongchan/attention-module#28]
hello，i would like to ask you that if you have the pre-train model of resnet101+CBAM? i added CBAM module to resnet-101 and  trained from scratch. But i have the problem of loss= Nan. Do i need to initialize the paramenters of this module?
